src/main/java/net/hypixel/nerdbot/util/watcher/rss/xmlparsers/SkyblockThreadParser.java

```diff
@@ -0,0 +1,53 @@
+package net.hypixel.nerdbot.util.watcher.rss.xmlparsers;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.log4j.Log4j2;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.parsers.SAXParser;
+import javax.xml.parsers.SAXParserFactory;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+
+@Log4j2
+public class SkyblockThreadParser {
+
+    public static HypixelThread getLastPostedSkyblockThread(String XML) {
+        try {
+            return parseSkyblockThreads(XML).get(0);
```

Comment:
Also make sure this is changed wherever it's used
```suggestion
    public static HypixelThread getLastPostedSkyBlockThread(String xml) {
        try {
            return parseSkyBlockThreads(xml).get(0);
```